tests/components/octoprint/test_binary_sensor.py

```diff
@@ -0,0 +1,81 @@
+"""The tests for Octoptint binary sensor module."""
+import asyncio
+import logging
+import unittest
+from unittest import mock
+
+from homeassistant.components.octoprint import binary_sensor as sensor
+from homeassistant.components.octoprint.const import DOMAIN
+from homeassistant.const import STATE_OFF, STATE_ON
+
+from tests.common import get_test_home_assistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TestOctoprintBinarySensorSetup(unittest.TestCase):
```

Comment:
All tests must be standalone pytest test functions instead of unittest class based.

Reply:
Converted